operations/mimir-mixin/dashboards/dashboard-utils.libsonnet

```diff
@@ -1206,6 +1242,24 @@ local utils = import 'mixin-utils/utils.libsonnet';
       replaceFields: replaceFields,
     }),
 
+  wrapMultiLabelReplace(query, labelReplaceArgSets=[{}])::
```

Comment:
it would be cool to define a type that says what goes in a `labelReplaceArgSet` but not sure if it's possible

Reply:
appears there are no user-defined types or object templates

Reply:
> it would be cool to define a type that says what goes in a `labelReplaceArgSet`

yeah, unfortunately in jsonnet everything is either a json literal or an object :( (or a function). 

If you want to make it easier to use, maybe naming it queryString might make it a tiny bit better.